web/src/components/data-fields/TaxClearanceCertificateAgencyDropdown.tsx

```diff
@@ -0,0 +1,54 @@
+import { MenuOptionSelected } from "@/components/MenuOptionSelected";
+import { MenuOptionUnselected } from "@/components/MenuOptionUnselected";
+import { TaxClearanceCertificateDataContext } from "@/contexts/taxClearanceCertificateDataContext";
+import {
+  getTaxClearanceCertificateAgencies,
+  LookupTaxClearanceCertificateAgenciesById,
+} from "@businessnjgovnavigator/shared";
+import { MenuItem, Select, SelectChangeEvent } from "@mui/material";
+import { ReactElement, ReactNode, useContext } from "react";
+
+export const TaxClearanceCertificateAgencyDropdown = (): ReactElement => {
+  const { state, setTaxClearanceCertificateData } = useContext(TaxClearanceCertificateDataContext);
+
+  const handleChange = (event: SelectChangeEvent): void => {
+    if (event.target.value) {
+      setTaxClearanceCertificateData({
+        ...state,
+        requestingAgencyId: event.target.value,
+      });
+    }
+  };
+
+  const renderValue = (value: string): ReactNode => {
+    return value === "" ? <></> : <>{LookupTaxClearanceCertificateAgenciesById(value).name}</>;
+  };
```

Comment:
[sand] can we break up the ternary?

```suggestion
    if (value === ""){
      return <></>
    }
     return <>{LookupTaxClearanceCertificateAgenciesById(value).name}</>;
  };
```